### PR TITLE
[docs] Add a disclaimer on parallelism

### PR DESCRIPTION
**Pull request by kciesielski**
I think this example can lead to problematic design, so it needs some sort of a disclaimer.

### CODE REVIEW

File: docs/src/main/paradox/consumer.md [hunk 1]
```diff
@@ -81,6 +81,8 @@ Scala
 Java
 : @@ snip [dummy](../../test/java/sample/javadsl/ConsumerExample.java) { #atLeastOnce }
 
+Important: The above example runs message processing and commit in the same `mapAsync` stage. This is fine as long as `parallelism` is set to 1. With higher parallelism it would be possible that some calls to `db.update` will be finished earlier than others and commit will be performed out of order. If you need higher parallelism, use separate stages for message processing and committing.
```
Comment (patriknw) [inline docs/src/main/paradox/consumer.md]: good point, should we rewrite the example to use separate stages?

Comment (kciesielski) [inline docs/src/main/paradox/consumer.md]: Good idea. I'll do this and update this PR.

**Comment by drewhk** [comment]
LGTM, good point!

**Comment by kciesielski** [comment]
PR updated. Now the Scala/Java examples use separate stages and there's a shorter disclaimer.

**Comment by patriknw** [comment]
LGTM

**Pull request merged**